tests/shared/helpers/notifyServer.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from "@playwright/test"
+
+export class NotifyServer {
+  private notifyBaseUrl = "https://notify.walletconnect.com"
+
+  public async sendMessage({
+    projectId,
+    projectSecret,
+    accounts,
+    url,
+    title,
+    body,
+    icon,
+    type
+  }: {
+    projectId: string,
+    projectSecret: string,
+    accounts: string[]
+    title: string,
+    body: string,
+    icon: string,
+    url: string
+    type: string
+  }) {
+    const request = JSON.stringify({
+      accounts,
+      notification: {
+	title,
+	body,
+	icon,
+	url,
+	type
+      }
+    })
+
+    const fetchUrl = `${this.notifyBaseUrl}/${projectId}/notify`
+
+    const headers = new Headers({
+      
+      Authorization: `Bearer ${projectSecret}`,
+      "Content-Type": "application/json"
+
+    })
+
+    const stream = new ReadableStream({
+      start(controller) {
+        controller.enqueue(new TextEncoder().encode(request));
+        controller.close(); // Close the stream after enqueueing the data
+      }
+    })
+
+    const fetchResults = await fetch(fetchUrl, {
+      method: "POST",
+      headers,
+      // @ts-ignore
+      duplex: true,
```

Comment:
What's this?

Reply:
Need to attach duplex streaming so it works on all browsers https://github.com/whatwg/fetch/issues/1254